aws/resource_aws_s3_access_point_test.go

```diff
@@ -477,6 +517,38 @@ resource "aws_s3_access_point" "test" {
 `, bucketName, accessPointName)
 }
 
+func testAccAWSS3AccessPointConfig_Bucket_Arn(rName string) string {
+	return fmt.Sprintf(`
+data "aws_outposts_outposts" "test" {}
+
+data "aws_outposts_outpost" "test" {
+  id = tolist(data.aws_outposts_outposts.test.ids)[0]
+}
+
+resource "aws_s3control_bucket" "test" {
+  bucket     = %[1]q
+  outpost_id = data.aws_outposts_outpost.test.id
+}
+
+resource "aws_vpc" "test" {
+  cidr_block = "10.0.0.0/16"
+
+  tags = {
+    Name = %[1]q
+  }
+}
+
+resource "aws_s3_access_point" "test" {
+  bucket = aws_s3_bucket.test.arn
```

Comment:
```suggestion
  bucket = aws_s3control_bucket.test.arn
```

Reply:
Great catch 🤦  (bummer not being able to actually run these)